dts/x86/intel/alder_lake.dtsi

```diff
@@ -258,6 +258,12 @@
 		compatible = "simple-bus";
 		ranges;
 
+		vtd: vtd@fed91000 {
```

Comment:
Don't enable it yet. (move that commit away).
I understand you needed that to build-test vtd on that platform, but ADL does not expose any IOAPIC rerouting capability. (which either could mean we need to move forward a 100% PCIe based system or... or I don't see)
So vtd is not going to work yet on ADL because we still use too much legacy stuff atm.